lib/elixir/lib/calendar/iso.ex

```diff
@@ -757,20 +748,41 @@ defmodule Calendar.ISO do
     if leap_year?(year), do: 1, else: 0
   end
 
+  defp days_to_year(days) when days < 0 do
+    year_estimate = -div(-days, @days_per_nonleap_year) - 1
+
+    {year, days_before_year} =
+      days_to_year(year_estimate, days, days_to_end_of_epoch(year_estimate))
+
+    leap_year_pad = if leap_year?(year), do: 1, else: 0
+    {year, leap_year_pad + 365 + days - days_before_year}
+  end
+
   defp days_to_year(days) do
-    year = Integer.floor_div(days, @days_per_nonleap_year)
+    year = div(days, @days_per_nonleap_year)
```

Comment:
Another minor nitpick: I like the `year_estimate` naming given in the function above, I would use the same naming here (also for consistency) 🙂